### DISCUSSION TITLE
KEDA HTTP Add-On Chart v0.3.1

### DISCUSSION THREAD

**Discussion by tomkerkhove** [kedacore/charts#357] [Announcements]
*Chart: v0.3.1 | App: 0.3.0*

Add-on for KEDA to scale HTTP workloads.

## TL;DR

```shell
helm repo add kedacore https://kedacore.github.io/charts
helm install keda kedacore/keda
helm install keda-add-ons-http kedacore/keda-add-ons-http
```

### Features

- Automatically add interceptor deployment name to scaler config (#251)
- Provide support for image pull secret in HTTP add-on (#296)

### Fixes / Changes

- Fix compatibility with Traefik (#286)

### Breaking Changes

None.

### Removal

None.

## New Contributors
* @AshutoshNirkhe made their first contribution in https://github.com/kedacore/charts/pull/239
* @bmalinconico made their first contribution in https://github.com/kedacore/charts/pull/236
* @marco2704 made their first contribution in https://github.com/kedacore/charts/pull/245
* @v-shenoy made their first contribution in https://github.com/kedacore/charts/pull/263
* @mac8005 made their first contribution in https://github.com/kedacore/charts/pull/255
* @Swalloow made their first contribution in https://github.com/kedacore/charts/pull/276
* @mhowell24 made their first contribution in https://github.com/kedacore/charts/pull/279
* @ph311o made their first contribution in https://github.com/kedacore/charts/pull/258
* @bmiguel-teixeira made their first contribution in https://github.com/kedacore/charts/pull/287
* @aslafy-z made their first contribution in https://github.com/kedacore/charts/pull/292
* @kvandenhoute made their first contribution in https://github.com/kedacore/charts/pull/296
* @joebowbeer made their first contribution in https://github.com/kedacore/charts/pull/305
* @erikkn made their first contribution in https://github.com/kedacore/charts/pull/312
* @christidis made their first contribution in https://github.com/kedacore/charts/pull/286
* @doctorpangloss made their first contribution in https://github.com/kedacore/charts/pull/308
* @StevenJDH made their first contribution in https://github.com/kedacore/charts/pull/319
* @schelv made their first contribution in https://github.com/kedacore/charts/pull/322
* @blottn made their first contribution in https://github.com/kedacore/charts/pull/332
* @kevinteng525 made their first contribution in https://github.com/kedacore/charts/pull/320
* @penghuazhou made their first contribution in https://github.com/kedacore/charts/pull/335
* @zerodayyy made their first contribution in https://github.com/kedacore/charts/pull/346
* @Shaked made their first contribution in https://github.com/kedacore/charts/pull/355
* @RomanBats made their first contribution in https://github.com/kedacore/charts/pull/353

**Full Changelog**: https://github.com/kedacore/charts/compare/keda-add-ons-http-v0.3.0...keda-add-ons-http-v0.3.1

<hr /><em>This discussion was created from the release <a href='https://github.com/kedacore/charts/releases/tag/keda-add-ons-http-v0.3.1'>KEDA HTTP Add-On Chart v0.3.1</a>.</em>
